packages/sandbox/src/file_watching_sandbox.ts

```diff
@@ -322,13 +326,7 @@ export class FileWatchingSandbox extends EventEmitter implements Sandbox {
       }
       return true;
     } catch (e) {
-      if (
-        e &&
-        typeof e === 'object' &&
-        'message' in e &&
-        typeof e.message === 'string' &&
-        e.message.includes('does not exist')
-      ) {
+      if (e instanceof ParameterNotFound) {
         return false;
       }
       // If we are unable to get the stack info due to other reasons(AccessDenied), we fail fast.
```

Comment:
we should clean up this comment as well.

Reply:
done